elasticsearch/sgconfig/action_groups.yml

```diff
@@ -45,6 +45,7 @@ READ:
 
 DELETE:
   - "indices:data/write/delete*"
+  - "indices:data/write/delete/byquery"
```

Comment:
@btaani I did not request to change this. Just drop a couple of lines about the rationale. If it doesn't hurt having the extra right on this action group, I am fine with it.

Reply:
I believe this is now outdated and GitHub incorrectly assings @periklis' comment to an updated commit (maybe not incorrectly, perhaps @btaani could have amended the commit instead, anyway....)

Reply:
Oh... I think GitHub has it correct. @btaani I would recommend you to squash all your commits into a single one and update the PR, then it will be clear and you will need to address Peri's comment.

Reply:
Yes, Peri's comment was on the previous commit.